examples/helm/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as helm from "@pulumi/kubernetes/helm";
+
+const nginx = new helm.v2.Chart("simple-nginx", {
+  // Represents chart `stable/nginx-lego@v0.3.1`.
+  repo: "stable", chart: "nginx-lego", version: "0.3.1",
+  values: {
+    // Override for the Chart's `values.yml` file. Zero out resource requests so it can be scheduled
+    // on our (wimpy) CI cluster.
+    nginx: {resources: null},
+    default: {resources: null},
+    lego: {resources: null},
```

Comment:
How does the equivalent look when done directly using Helm?  Is this `null` thing something that is Pulumi-specific, or is there a model for this directly with Helm `values.yml`?

Reply:
This is how you "unset" values in `values.yml` as far as I can tell. This isn't Pulumi-specific.

Reply:
Got it.  https://github.com/helm/helm/blob/master/docs/chart_template_guide/values_files.md#deleting-a-default-key